vaadin-icons-flow-integration-tests/src/test/java/com/vaadin/flow/component/icon/tests/IconIT.java

```diff
@@ -37,14 +37,14 @@ public void init() {
 
     @Test
     public void basicIcons() {
-        assertIconProperty("edit-icon", "edit");
-        assertIconProperty("close-icon", "close");
+            assertIconProperty("close-icon", "vaadin", "close");
```

Comment:
Looks like each changed line in this file has extra indentation. Please fix those. Otherwise LGTM.